src/views/MainView.vue

```diff
@@ -66,3 +67,9 @@ export default Vue.extend({
   }
 })
 </script>
+
+<style scoped>
+  .hr{
+    height: 4px;
```

Comment:
use rems ? and or use theme style sheet for margins to get consistent margins

Reply:
moved to external style